FCCSW_ecal/write_calibration_xml.py

```diff
@@ -58,44 +65,68 @@
                 if subnode.localName == 'readout' and subnode.getAttribute('name') == 'ECalBarrelPhiEta':
                     for subsubnode in subnode.childNodes:
                         if subsubnode.localName == 'segmentation':
-                            n_phi_bins = subsubnode.getAttribute('phi_bins')
+                            n_phi_bins = int(subsubnode.getAttribute('phi_bins'))
```

Comment:
Is this script really working for k4geo/FCCee/ALLEGRO/compact/ALLEGRO_o1_v02/ECalBarrel_thetamodulemerged.xml ?

Reply:
this script is used to update all .py - the particular piece of code you are pointing go is only for a readout of type ECalBarrelPhiEta so it won't affect the code related to the inclined readout class. Here I just added the int(..) part because phi_bins before could be a string like 1536/2, and without the int, n_phi_bins would be treated as a string rather than as a number